src/number_theory/modular_forms/basic.lean

```diff
@@ -0,0 +1,286 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck, David Loeffler
+-/
+import algebra.module.submodule.basic
+import analysis.complex.upper_half_plane.basic
+import linear_algebra.general_linear_group
+import linear_algebra.special_linear_group
+import algebra.direct_sum.ring
+import number_theory.modular
+import geometry.manifold.mfderiv
+import analysis.complex.upper_half_plane.functions_bounded_at_infty
+import number_theory.modular_forms.slash_actions
```

Comment:
You can probably shorten this list a lot.  Comment each one in turn and see if it breaks, then see if any of the remaining ones can be replaced by their imports, rinse and repeat.